src/lib/components/submit/SubmitWrite.svelte

```diff
@@ -21,6 +21,8 @@
 	<SubmitMotion {metadata} {content} />
 {:else if proposalAction === 'AddOrRemoveNodeProvider'}
 	<SubmitAddNodeProvider />
+{:else}
+	<SubmitMotion {metadata} {content} />
```

Comment:
I would rather like that we don't duplicate usage of components that way properties are binded for one use case